<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>800</height>
    </rect>
   </property>
   <widget class="QLabel" name="Background_Label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>800</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Home_Button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Home_Button
{
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
border : 0px solide rgba(0,0,0,0)
}
QPushButton#Home_Button:hover
{
background-color: rgb(62, 168, 234);
   border : 3px solide rgba(0,0,0,0);
   border-radius: 5px;
}
QPushButton#Home_Button:pressed
{
    
	color: rgb(0, 0, 0);
	background-color: rgb(62, 168, 234);
   border : 3px solide rgba(0,0,0,0);
   border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AboutUs_Button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#AboutUs_Button
{
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
border : 0px solide rgba(0,0,0,0)
}
QPushButton#AboutUs_Button:hover
{
background-color: rgb(62, 168, 234);
   border : 3px solide rgba(0,0,0,0);
   border-radius: 5px;
}
QPushButton#AboutUs_Button:pressed
{
    
	color: rgb(0, 0, 0);
	background-color: rgb(62, 168, 234);
   border : 3px solide rgba(0,0,0,0);
   border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>About Us</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Contact_Button">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>20</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Contact_Button
{
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
border : 0px solide rgba(0,0,0,0)
}
QPushButton#Contact_Button:hover
{
background-color: rgb(62, 168, 234);
   border : 3px solide rgba(0,0,0,0);
   border-radius: 5px;
}
QPushButton#Contact_Button:pressed
{
    
	color: rgb(0, 0, 0);
	background-color: rgb(62, 168, 234);
   border : 3px solide rgba(0,0,0,0);
   border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>Contact</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Join_Button">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family> Abdoullah Ashgar EL-kharef</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Join_Button
{
background-color: rgb(255, 255, 255);
border-radius:12px;
}


QPushButton#Join_Button:hover
{
	
	
	background-color: rgb(62, 168, 234);

}

QPushButton#Join_Button:pressed
{
background-color:rgb(255, 255, 255);

}



</string>
    </property>
    <property name="text">
     <string>Join Now</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Registere_Button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>680</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family> Abdoullah Ashgar EL-kharef</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Registere_Button
{
background-color: rgb(255, 255, 255);
border-radius:12px;
}


QPushButton#Registere_Button:hover
{
	
	
	
background-color: rgb(62, 168, 234);

}

QPushButton#Registere_Button:pressed
{
background-color:rgb(255, 255, 255);

}



</string>
    </property>
    <property name="text">
     <string>Registere</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
